app/views/ForwardLivechatView.tsx

```diff
@@ -85,11 +122,14 @@ const ForwardLivechatView = ({ forwardRoom, navigation, route, theme }) => {
 	};
 
 	useEffect(() => {
+		navigation.setOptions({
+			title: I18n.t('Forward_Chat')
+		});
 		getRoom();
 	}, []);
 
 	useEffect(() => {
-		if (room) {
+		if (!isEmpty(room)) {
```

Comment:
I changed this line to working properly with the `useState({})`

Reply:
revert this change please :)